Makefile

```
@@ -14,6 +14,8 @@ help:
 	@echo "    dev-prerequisites:  Install necessary development prerequisites on your system"
 	@echo "    push-check:         Replicates the github workflow checks as close as possible"
 	@echo "                        (do this before pushing!)"
+	@echo "	   coverage-report:    Run unit tests and generate an HTML coverage report."
+	@echo "    coverage-dump:     Run unit tests and display function-level coverage information."
```

tiny nit-pick? but I'll approve anyway
```suggestion
	@echo "    coverage-dump:      Run unit tests and display function-level coverage information."
```